src/overrideFetch.ts

```diff
@@ -42,7 +42,7 @@ export function overrideFetch() {
       clearCurrentNetmockReplyTrace();
       if (!(res instanceof NetmockResponse)) {
         if (replyTrace) {
-          throw getErrorWithCorrectStack('Error: detected unreturned reply. Did you used "reply()" instead of "return reply()"?', replyTrace);
+          // throw getErrorWithCorrectStack('Error: detected unreturned reply. Did you used "reply()" instead of "return reply()"?', replyTrace);
```

Comment:
is it partly working? if so maybe  it make sense to console.warn instead

Reply:
it is not reliable, I will fix it in a separate jira when i'll have the time.